src/commcare_cloud/ansible/roles/nginx/vars/cchq_ssl.yml

```diff
@@ -11,7 +11,9 @@ nginx_sites:
     - name: "{{ deploy_env }}_formplayer"
       hosts: formplayer
       port: "{{ formplayer_port }}"
-      method: "hash $cookie_sessionid consistent"
+      # hash off sessionid cookie or X-FORMPLAYER-SESSION header
+      # but only one of the two will be present in any given request
+      method: "hash $cookie_sessionid$http_x_formplayer_session consistent"
```

Comment:
This is using the nginx $http_* variable syntax for headers: http://nginx.org/en/docs/http/ngx_http_core_module.html#var_http_